### DISCUSSION TITLE
Code samples from readme not working.

### DISCUSSION THREAD

**Discussion by stephan-v** [spatie/laravel-twitter-streaming-api#46] [Q&A]
The examples of the public stream imply there is a user key in the tweet array:

```
$tweet['user']
```

Yet when dump the actual `$tweet` array there is no such key in there.

```
[
  "data" => [
    "id" => "1337"
    "text" => "text"
  ]
  "matching_rules" => [
    0 => [
      "id" => "1111"
      "tag" => "#thecodeexampleseemswrong"
    ]
  ]
]
```

I am simply copy pasting the exact sample from the readme for the public stream. Has twitter updated their API or does something else need to happen to actually fetch the user data as well?

**Comment by tomswinkels** [comment]
Same issue..

**Comment by simonbuehler** [comment]
you guys should use it per [docs ](https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/api-reference/get-tweets-search-stream )now: 

```php
TwitterStreamingApi::publicStream()
               ->whenHears('#php', function (array $tweet) {
                   dump($tweet);
               })->startListening(
    new Sets(
        new Fieldset('expansions', 'author_id'),
        new Fieldset('user.fields', 'profile_image_url')
    )
);
``` 

@AlexVanderbist maybe add this to the readme?

**Comment by felixdorn** [comment]
Yeah, there's also a good bit about how to get more data per tweet here, https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/api-reference/get-tweets-search-stream.

Also please consider #43.
